firewall/handlers/blacklist_handler.go

```diff
@@ -24,45 +24,43 @@ func(handler * BlacklistHandler) CheckQuery(query string) error {
 		}
 	}
 
-
 	//Check tables
 	if len(handler.tables) != 0 {
 		parsedQuery, err := sqlparser.Parse(query)
 		if err != nil {
-			return err
+			return errors.New(err.Error() + " | blacklist (tables)")
```

Comment:
bad pattern to concatenate strings via "+" operator. And bad way to create error in such way. Better to add new type of error. Then we can easily check is it correct error type in unit-tests without string comparison.

Reply:
Agree. Refactored